docs/en/test.md

```diff
@@ -8,10 +8,20 @@ To test nas method, you can use following command
 python tools/${task}/test_${task}.py ${CONFIG_FILE} ${CHECKPOINT_PATH} --cfg-options algorithm.mutable_cfg=${MUTABLE_CFG_PATH} [optional arguments]
 ```
 
+- `task`: one of ``mmcls``、``mmdet`` and ``mmseg``
 - `MUTABLE_CFG_PATH`: Path of `mutable_cfg`. `mutable_cfg` represents **config for mutable of the subnet searched out**, used to specify different subnets for testing. An example for `mutable_cfg` can be found [here](https://github.com/open-mmlab/mmrazor/blob/master/configs/nas/spos/SPOS_SHUFFLENETV2_330M_IN1k_PAPER.yaml).
 
 The usage of optional arguments are the same as corresponding tasks like mmclassification, mmdetection and mmsegmentation.
 
+For example,
+
+<pre>
+python tools/mmcls/test_mmcls.py \
+  configs/nas/spos/spos_subnet_shufflenetv2_8xb128_in1k.py \
+  <em>your_retraining_checkpoint_path</em> \
```

Comment:
NAS algorithms like once-for-all or BigNAS may not need to retrain subnet.
```suggestion
  <em>your_subnet_checkpoint_path</em> \
```

Reply:
Fixed, thx!